nightwatch/test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+ENV=$1
+[ -z "$ENV" ] && ENV='default'
+
+if [ "$ENV" != "default" ] ; then
+    CRED="../mycredentials"
+    [ ! -f $CRED ] && echo "Could not find credentials file: $CRED" && exit 1
+    CRED="env `cat $CRED`"
```

Comment:
sure, my point is why do you need to run it at all? all you'll do is redeclare a load of env variables that it already gets anyawy